published/Todoist/items/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "publishedId" : "iCwW43WYQzqo_njXaBfd_A",
+  "requiredOAuth2Applications" : [ {
+    "applicationId" : "tRIrUQTfQ7qw-roVvQeg_g",
```

Comment:
Aha these won't work. Need to specify something else in exports, like API and scope required